tests/components/garmin_connect/test_config_flow.py

```diff
@@ -93,14 +97,3 @@ async def test_unknown_error(hass, mock_garmin_connect):
     )
     assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
     assert result["errors"] == {"base": "unknown"}
-
-
-async def test_abort_if_already_setup(hass, mock_garmin_connect):
```

Comment:
Why do we remove this test?

Reply:
I forgot to rewrite it I guess, will have a look.

Reply:
Added the test